e2e/dractions/retry.go

```diff
@@ -147,6 +157,15 @@ func getCurrentCluster(client client.Client, namespace string, placementName str
 	return clusterName, nil
 }
 
+// return dr cluster client
+func getDRClusterClient(clusterName string, drpolicy *ramen.DRPolicy) client.Client {
+	if clusterName == drpolicy.Spec.DRClusters[0] {
```

Comment:
Is it possible that DRClusters be in random order?

Reply:
these clients are built from the config yaml with format below:

```
Clusters:
  c1:
    kubeconfigpath: /home/par/.config/drenv/rdr-rdr/kubeconfigs/rdr-dr1
  c2:
    kubeconfigpath: /home/par/.config/drenv/rdr-rdr/kubeconfigs/rdr-dr2
  hub:
    kubeconfigpath: /home/par/.config/drenv/rdr-rdr/kubeconfigs/rdr-hub
```

And in the drpolicy, it is created by drenv with following content:
```
spec:
    drClusters:
        - rdr-dr1
        - rdr-dr2
```

so as long as drpolicy cluster list does not change sequence (which I believe will not change) and maps correctly to the config yaml, we should be fine.